onebusaway-android/src/main/AndroidManifest.xml

```diff
@@ -268,6 +276,16 @@
                 <action android:name=".nav.NavigationReceiver" />
             </intent-filter>
         </receiver>
+        <receiver
+            android:name=".nav.FeedbackReceiver"
+            android:enabled="true"
+            android:exported="true" >
```

Comment:
We don't want other apps to be able to see this receiver, so let's remove `android:exported="true"`

Reply:
Removed `android:exported="true"`.